app/assets/javascripts/ga_cross_domain_form_tracking.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-env jquery */
+/* global ga */
```

Comment:
Stole this from [here](https://github.com/alphagov/govuk_publishing_components/blob/master/app/assets/javascripts/govuk_publishing_components/analytics/ecommerce.js#L2)

Is this the right way to do it?

Reply:
Not sure this is really needed - does the linter really throw an error?

Reply:
Yeah if I remove it, it gets picked up on CI

Reply:
Yeah reconfirmed if you run `govuk-docker-run yarn lint" you can see the error if you remove that line